frontend/src/styles/UniversalLandingPage.module.scss

```diff
@@ -0,0 +1,46 @@
+.topDesignLove {
+  justify-items: center;
+  position: absolute;
+  width: 1440;
+  height: 340;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  fill: none;
+}
+
+.nightlineDesignLeft {
+  position: absolute;
+  width: 543.62px;
```

Comment:
You might want to use rem instead of px from now on to be responsive to different screen sizes